### PR TITLE
Custos accidentals as attributes. Custos accidentals rendered

### PR DESCRIPTION
**Pull request by davidrizo**

### CODE REVIEW

**Review by rettinghaus — APPROVED** [review]

File: src/iomei.cpp [hunk 1]
```diff
@@ -5031,6 +5031,20 @@ bool MEIInput::ReadCustos(Object *parent, pugi::xml_node custos)
...
+    AttAccidentalGestural accidentalGestural;
+    accidentalGestural.ReadAccidentalGestural(custos);
+    if (accidental.HasAccid() || accidentalGestural.HasAccidGes()) {
+        Accid *vrvAccid = new Accid();
+        vrvAccid->IsAttribute(true);
+        vrvAccid->SetAccid(accidental.GetAccid());
+        vrvAccid->SetAccidGes(accidentalGestural.GetAccidGes());
+        vrvCustos->AddChild(vrvAccid);
+    }
+    // -- end copy
```
Comment (lpugin) [inline src/iomei.cpp:5046]: Could you make a common `MEIInput::ReadAccidAttr(pugi::xml_node node, Object *object)` to be place there https://github.com/rism-digital/verovio/blob/7a392176fc9d49b97c0c6fac555113e7ffc0311e/include/vrv/iomei.h#L692-L701 ? Thanks

File: src/view_element.cpp [hunk 2]
```diff
@@ -718,6 +718,9 @@ void View::DrawCustos(DeviceContext *dc, LayerElement *element, Layer *layer, St
     }
 
     dc->EndGraphic(element, this);
+
+    /************ Draw children (accidentals, etc) ************/
+    DrawLayerChildren(dc, custos, layer, staff, measure);
```
Comment (lpugin) [inline src/view_element.cpp:723]: Drawing the children should be done before ending the graphic. Otherwise the SVG tree will not match the MEI one

**Review by lpugin — CHANGES_REQUESTED** [review]
Thanks! A few changes requested

File: src/view_element.cpp (hunk 2 above)
Comment (davidrizo) [reply src/view_element.cpp:723]: Thanks for the comments. I've added those changes

File: src/iomei.cpp [hunk 3]
```diff
@@ -5346,7 +5348,8 @@ bool MEIInput::ReadNote(Object *parent, pugi::xml_node note)
         vrvAccid->SetAccid(accidental.GetAccid());
         vrvAccid->SetAccidGes(accidentalGestural.GetAccidGes());
         vrvNote->AddChild(vrvAccid);
-    }
+    }*/
```
Comment (lpugin) [inline src/iomei.cpp:5351]: You can remove this

**Review by lpugin — CHANGES_REQUESTED** [review]
Don't forget to run clang-format https://github.com/rism-digital/verovio/blob/master/doc/guidelines.md

File: src/iomei.cpp (hunk 3 above)
Comment (davidrizo) [reply src/iomei.cpp]: Done

**Comment by davidrizo** [comment]
Requested changes applied

**Pull request merged**